spec/integration/data_stream/fields/fields.spec.yml

```diff
@@ -613,7 +613,7 @@ spec:
 
 # JSON patches for newer versions should be placed on top
 versions:
-  - before: 3.4.0
+  - before: 3.5.0
     patch:
       - op: remove
         path: "/items/properties/type/enum/35" #remove semantic_text type
```

Comment:
Issue open to ensure we get this done at some point https://github.com/elastic/kibana/issues/220294